### PR TITLE
DP-33490 - Setup Percy with Browser Automate

### PR DESCRIPTION
**Pull request by tom-fleming**
**Description:**
Adds Percy w/ Browser Automate.

See README: https://github.com/massgov/openmass/pull/2551/files#diff-f19d19a7f519de4434ef52d31ab8eda0ff9f7bca9ed6d1a618619a1ede41bfc0

If you find you need access to view Percy output or BrowserStack Automate output, reach out to @tom-fleming or @joeg8612 

**Jira:** (Skip unless you are MA staff)
DP-33490

**To Test:**
- [ ] Run a custom comparison with `drush ma:ci:percy --target=test --list=post-release --ci-branch=feature/DP-33490_setup-percy-browser-automate`

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by joeg8612** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against eebdfa4fa1fcea0f3c8be45b4cceae725789aaf6
</p>

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2be0cf533d310caf329b63f2342ad45187c312d7
</p>

File: changelogs/DP-33490.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Added:
+  - description: Added Percy for visual regression testing.
```
Comment (tom-fleming) [inline changelogs/DP-33490.yml]:
```suggestion
  - description: Added Percy configuration for visual regression testing.
```

File: changelogs/DP-33490.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Added:
+  - description: Added Percy configuration for visual regression testing.
```
Comment (tom-fleming) [inline changelogs/DP-33490.yml]:
```suggestion
  - description: Added Percy configurations for visual regression testing.
```

**Pull request merged**